dev/NavigationView/NavigationView.cpp

```diff
@@ -869,6 +1004,10 @@ winrt::IndexPath NavigationView::GetIndexPathForContainer(const winrt::Navigatio
         path.insert(path.begin(), parentIR.GetElementIndex(child.try_as<winrt::UIElement>()));
     }
 
+    isInFooterMenu = parent == m_leftNavFooterMenuRepeater.get() || parent == m_topNavFooterMenuRepeater.get();
+
+    path.insert(path.begin(), isInFooterMenu ? 1 : 0);
```

Comment:
nit: I think it might be clearer to define constants in order to indicate footer vs main and use them throughout the code.